drivers/syslog/vsyslog.c

```diff
@@ -212,6 +213,15 @@ int nx_vsyslog(int priority, FAR const IPTR char *fmt, FAR va_list *ap)
   ret += lib_sprintf(&stream.public, "%s", CONFIG_SYSLOG_PREFIX_STRING);
 #endif
 
+#if CONFIG_TASK_NAME_SIZE > 0 && defined(CONFIG_SYSLOG_PROCESS_NAME)
+  /* Pre-pend the Process ID */
```

Comment:
```suggestion
  /* Prepend the Process ID */
```
Please, could you fix this typo? It is repeated several times in this file.

Reply:
Done